apps/web/app/features/stacking/start-pooled-stacking.tsx

```diff
@@ -161,29 +230,58 @@ function StartPooledStackingLayout({ poolName }: StartPooledStackingLayoutProps)
               <Stack gap="space.02">
                 <StackingFormItemTitle title="Address to receive rewards" />
                 <ChooseRewardsAddress />
+                <styled.span textStyle="caption.01" color="ink.text-subdued">
+                  This is where the pool will deposit your rewards each cycle.
+                </styled.span>
               </Stack>
 
+              <Divider />
+
               <Stack gap="space.02">
                 <StackingFormItemTitle title="Duration" />
                 <ChoosePoolingDuration />
               </Stack>
 
+              <Divider />
+
               <Stack gap="space.02">
                 <StackingFormItemTitle title="Details" />
                 <PoolingDetails poolAddress={poolStxAddress} contractAddress={poxWrapperContract} />
               </Stack>
 
+              <Divider />
+
               <Stack gap="space.02">
                 <StackingFormItemTitle title="Pooling conditions" />
                 <ChoosePoolingConditions />
               </Stack>
             </Stack>
           </Form>
         }
-        stackingInfoPanel={
-          <StackingFormInfoPanel>
-            <StackingStepsCard poolAmount="999,999.99" />
-          </StackingFormInfoPanel>
+        stackingStepsPanel={
+          <StackingFormStepsPanel>
+            <StackingStepsCard
+              onSubmit={onSubmit}
+              confirmationState={{
+                terms: {
+                  accepted: termsConfirmed,
+                  loading: false,
+                  visible: true,
+                },
+                'interact-wallet': {
```

Comment:
if we need to use these values in objects, perhaps camelcase is better

Reply:
yeah, used camelcase